microshift_install/microshift-install-rpm.adoc

```diff
@@ -1,8 +1,62 @@
 :_content-type: ASSEMBLY
 [id="microshift-install-rpm"]
-= Installing MicroShift from RPM
+= Installing MicroShift from RPM package
 include::_attributes/attributes-microshift.adoc[]
 :context: microshift-install-rpm
 toc::[]
 
-// The content of this file should be based on https://github.com/openshift/microshift/blob/main/docs/devenv_rhel8.md but without the steps for building from source.
+You can install MicroShift from an RPM package on a machine with {op-system-first} {op-system-version}. 
+
+To embed MicroShift in a {op-system} image, see xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Embedding MicroShift into a {op-system} image]
+
+[id="system-requirements-installing-microshift"]
+== System requirements for installing MicroShift 
```

Comment:
I think these are more-or-less the same for rpm and rpm-ostree systems. Should that be factored out into a common module?

Reply:
Done